src/data/accessibility-quiz.ts

```diff
@@ -29,6 +29,16 @@ const accessibilityQuiz = [
       "The section element holds semantic meaning and defines sections within the HTML document.",
     Link: "https://www.freecodecamp.org/news/semantic-html5-elements/",
   },
+  {
+    Question: "Which of the following reason is correct? A website should have a good contract ratio between foreground and background because",
+    Answer: "people with moderately low vision can read and access the website contents.",
+    Distractor1: "good contrast ratio will improve the entire web design.",
+    Distractor2: "secreen reader can distinguish better between foreground text and background.",
```

Comment:
```suggestion
    Distractor2: "screen readers can distinguish better between foreground and background text.",
```

Reply:
I think we should do something like this `foreground and background colors` instead of "`foreground and background text`. This is because the background is just colors most of the time. Usually, there're foreground texts and background colors. 

What do you think about this?